helm/demo.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
This is great! Please include a description in the PR talking about this script, including how to use it and how to clean up at the end:
```
$ helm ls
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
ovn-kubernetes  default         1               2024-02-27 21:17:53.349478111 +0000 UTC deployed        ovn-kubernetes-0.1.0    1.16.0

$ helm show values ./ovn-kubernetes | grep repository
    # -- Image repository for ovn-kubernetes components
    repository: ghcr.io/ovn-org/ovn-kubernetes/ovn-kube-u


To clean up, do:
$ helm uninstall ovn-kubernetes
release "ovn-kubernetes" uninstalled
$ kind delete cluster --name ovn-helm
```

Maybe instead of `demo.sh`, let's call it: `basic-deploy.sh`

Reply:
sounds good, thx.